app/containers/ItunesContainer/reducer.js

```diff
@@ -0,0 +1,36 @@
+/*
+ *
+ * HomeContainer reducer
```

Comment:
Make use of react floki

Reply:
Sorry i didn't got it, can you elaborate